scripts/deploy/runner.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+
+set -e
+
+if [[ -z "$1" ]] || [[ -z "$2" ]]; then
+  echo "PROJECT_ID and SUBMISSION_BUCKET_NAME are mandatory"
+  exit 1
+fi
+
+PROJECT_ID=$1
+SUBMISSION_BUCKET_NAME=$2
+IMAGE_TAG="${3:-latest}"
+
+helm tiller run \
+    helm upgrade --install \
+    survey-runner \
+    helm/runner \
+    --set projectId=${PROJECT_ID} \
```

Comment:
As mentioned in your other PR I think we can remove the need for the `projectId` to be passed in as it defaults to your current project